productList.html

```diff
@@ -77,6 +81,134 @@ <h2 class="main-title">
+        <button class="nav-close-btn"><span class="material-icons icon"> close </span></button>
+
+      </div>
+      <hr class="hr-line">
+      <div class="navbar-item-container secondary-color">
+        <a href="./index.html" class="secondary-color">
+          <div class="navbar-unit flex">
+            <span class="material-icons nav-icon"> home </span>
+            <div class="navbar-unit-title">Home</div>
+          </div>
+        </a>
+        <a href="./productList.html" class="secondary-color">
+          <div class="navbar-unit flex">
+            <span class="material-icons nav-icon"> shopping_cart </span>
+            <div class="navbar-unit-title">Shop Now</div>
+          </div>
+        </a>
+        <div class="hide-mini-desktop">
+          <a href="./login.html" class="secondary-color">
+            <div class="navbar-unit flex">
+              <span class="material-icons nav-icon"> login </span>
+              <div class="navbar-unit-title">Login</div>
+            </div>
+          </a>
+          <a href="./wishlist.html" class="secondary-color">
+            <div class="navbar-unit flex">
+              <span class="material-icons nav-icon"> favorite </span>
+              <div class="navbar-unit-title">Wishlist</div>
+            </div>
+          </a>
+          <a href="./cart.html" class="secondary-color">
+            <div class="navbar-unit flex">
+              <span class="material-icons nav-icon"> shopping_cart </span>
+              <div class="navbar-unit-title">Cart</div>
+            </div>
+          </a>
+        </div>
+      </div>
+    </nav>
+
+<!-- filter in mobile view -->
+    <div class="filter-mobile-view">
+      <header class="header-filter-mb flex justify-between align-center">
+        <div class="mb-filter filter-header flex">
+          <h3 class="primary-color">Filters</h3>
+          <p class="secondary-color clear-link">Clear</p>
+        </div>
+        <span class="material-icons filter-dropdown">
+          arrow_drop_down
+          </span>
+          <span class="material-icons filter-dropup hide-nav">
+            arrow_drop_up
+            </span>
+      </header>
+      <section class="filter-mobile-view-container">
+        <div class="filter-mob-container hide-nav">
+          <hr class="rg-line">
+          <div class="mb-filter filter-content">
+            <div class="filter-inner-content">
+              <h3 class="mb-filter filter-content-title">PRICE</h3>
+              <div class="filter-item align-center">
+                <input type="range" min="1" max="100" value="50">
+              </div>
+            </div>
+    
+            <hr class="rg-line" />
+    
+            <div class="filter-inner-content">
+              <h3 class="mb-filter filter-content-title secondary-color">CATEGORY</h3>
+              <div class="mb-filter filter-item align-center">
+                <input type="checkbox">
```

Comment:
you have used closing/tag for input before, so continue using it throughout the code